db/migrate/20170530173726_add_filepath_to_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddFilepathToSubmissions < ActiveRecord::Migration[5.1]
+  def change
+    add_column :submissions, :filepath, :string
+  end
+end
```

Comment:
Yeah, that's what I mean. We don't actually have production data to worry about.